pandas/core/ops.py

```diff
@@ -731,7 +700,9 @@ def safe_na_op(lvalues, rvalues):
                                                   lambda x: op(x, rvalues))
             raise
 
-    def wrapper(left, right, name=name, na_op=na_op):
+    def wrapper(left, right, na_op=na_op):
+        # TODO: `na_op` does not belong in Series.__{op}__ signature.  But this
```

Comment:
huh?

Reply:
0.21.1:
```
>>> inspect.getargspec(pd.Series.__add__)
ArgSpec(args=['left', 'right', 'name', 'na_op'], varargs=None, keywords=None, defaults=('__add__', <function na_op at 0x10c4b7d70>))
```

Reply:
and so what?

Reply:
... and there's [precedent](https://github.com/pandas-dev/pandas/pull/17179#discussion_r132919572) for caring about signatures that make sense, especially in super-commonly-used user-facing methods.

Reply:
Don't worry about it anyway, I found a way to get na_op and name out of the signature.